docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -210,7 +233,75 @@ data:
 EOF
 ----
 +
-The resources in the `ConfigMap` contain cell-specific configurations.
+
+. Create dataplane services for Nova cells to enable pre-upgrade workarounds,
+and to configure the {compute_service} services for a picked storage back end:
+
++
+[source,yaml]
+----
+for CELL in $(echo $RENAMED_CELLS); do
+  oc apply -f - <<EOF
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-$CELL
+spec:
+  dataSources: <1>
+    - secretRef:
+        name: nova-$CELL-compute-config <2>
+    - secretRef:
+        name: nova-migration-ssh-key <3>
+    - configMapRef:
+        name: nova-cells-global-config
+        optional: true
+  playbook: osp.edpm.nova
+  caCerts: combined-ca-bundle
+  edpmServiceType: nova
+  containerImageFields:
+  - NovaComputeImage
+  - EdpmIscsidImage
+EOF
+  done
+----
++
+
+* If TLS Everywhere is enabled, append the following content to the `OpenStackDataPlaneService` CR:
++
+[source,yaml]
+----
+  tlsCerts:
+    contents:
+      - dnsnames
+      - ips
+    networks:
+      - ctlplane
+    issuer: osp-rootca-issuer-internal
+  caCerts: combined-ca-bundle
+  edpmServiceType: nova
+----
++
+<1> To enable a local metadata services for a cell<N>, append a `spec.dataSources.secretRef` to reference
+an additional auto-generated `nova-cell<N>-metadata-neutron-config` secret. You should have also set
+`spec.nova.template.cellTemplates.cell<N>.metadataServiceTemplate.enable` in the `OpenStackControlPlane/openstack` CR.
+<2> The secret `nova-cell<N>-compute-config` auto-generates for each `cell<N>`.
```

Comment:
Can we backlink to the control plane adoption doc explaining how to create the cells on the control plane first?